examples/operator/101_initial_cluster.yaml

```diff
@@ -27,25 +29,58 @@ spec:
       resources:
         requests:
           cpu: 100m
-          memory: 128Mi
+          memory: 256Mi
         limits:
-          memory: 512Mi
+          memory: 256Mi
```

Comment:
There are changes to requests/limits of the various components in this file. I understand needing to increase the request values. What is the reason for lowering the limits?

Reply:
I didn't look too much into that, just copied over the configs from the operator example on VTop. I can revert changes to these if you like

Reply:
Should be fine to copy over, but let's make sure to make them uniform across all the yaml files. It looks like the later steps request more (which can lead to unnecessary pod restarts). Just copy over the values from 302 to all the files.

Reply:
You got it.

Reply:
Verified that they are already in sync. There are no unnecessary restarts in the example.

Reply:
They aren't in sync on `main` 🤣